src/actions/action_select_unit.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios'
+
+import { UNIT_SELECTED } from '../constants/action_types'
+import { UNIT_DETAIL_URL } from '../constants/api'
+
+export function selectUnit (unit) {
+  const request = axios.get(`${UNIT_DETAIL_URL}${unit.unitCode}`)
```

Comment:
Even though our API guarantees that every unit exists in all API calls.

You should try and look at handling errors, such as Server Errors, Not Found Errors, these have the status codes:

- 4xx: Client Error
    - 404: Not Found
- 5xx: Server Error